### DISCUSSION TITLE
wants me to install Ciscure Secure Client in order to start, although it is already installed

### DISCUSSION THREAD

**Discussion by lweilg** [neurodesk/neurodesk.github.io-hugo#457] [Q&A]
When starting the session, it asks me to install the Cisco Secure Client, which I have already installed and activated (tried it multiple times as well). I cannot move forward.

**Comment by stebo85** [comment]
We are not installing this - so it must be something from your institution? When is this request appearing? Can you provide screenshots?

**Reply by lweilg** [reply]
![552182A8-8E41-4E7D-B004-C089334DD345](https://github.com/user-attachments/assets/f993af9b-d6b5-4d52-8bee-c5e079cfa977)
It appears after I connect to my remote Neurodesk server.

**Reply by stebo85** [reply]
Yes, that's not from us. Where is your Neurodesk server hosted and who set it up?

**Reply by stebo85** [reply]
Dear @lweilg - is it working for you now? Do you otherwise want to schedule a quick a zoom meeting to look at this together?

**Comment by lweilg** [comment]
Yes, sorry. I think I have figured it out now. Thank you!

Kind regrads,

**Comment by stebo85** [comment]
Great to hear . What was the problem?


Thank you

Steffen

--

W: https://mri.sbollmann.net<https://mri.sbollmann.net/>  | W: https://www.neurodesk.org<https://www.neurodesk.org/>  | T: https://twitter.com/sbollmann_MRI  | G: https://github.com/stebo85

Book meeting: https://calendly.com/s-bollmann/meeting

________________________________
